pkg/operator/csi/csicontrollerset/csi_controller_set.go

```diff
@@ -116,11 +116,14 @@ func (c *CSIControllerSet) WithStaticResourcesController(
 func (c *CSIControllerSet) WithCredentialsRequestController(
 	name string,
 	operandNamespace string,
-	assetFunc func(string) []byte,
+	assetFunc resourceapply.AssetFunc,
 	file string,
 	dynamicClient dynamic.Interface,
 ) *CSIControllerSet {
-	manifestFile := assetFunc(file)
+	manifestFile, err := assetFunc(file)
+	if err != nil {
+		panic("asset: Asset(" + file + "): " + err.Error())
```

Comment:
I think you can just use %q and %v instead of concatenating

Reply:
Agree. Initially I copied this part from go-bindata generated code: https://github.com/openshift/openstack-cinder-csi-driver-operator/blob/master/pkg/generated/bindata.go#L1296 to make it the same.

I'll fix it in a minute.